vendor/bat-native-ads/src/bat/ads/internal/ads_history/ads_history.h

```diff
@@ -22,17 +22,7 @@ struct PromotedContentAdInfo;
 
 namespace history {
 
-// Maximum entries based upon 7 days of ad history for
-// |kAdNotificationsPerDayFrequencyCap| ad notifications per day with 2
-// confirmation types (viewed and either clicked or dismissed) and
-// |kNewTabPageAdsPerDayFrequencyCap| new tab page ads per day with 2
-// confirmation types (viewed and clicked)
-// |kPromotedContentAdsPerDayFrequencyCap| promoted content ads per day with 2
-// confirmation types (viewed and clicked)
-const size_t kMaximumEntries =
-    7 * ((kAdNotificationsPerDayFrequencyCap * 2) +
-         (kNewTabPageAdsPerDayFrequencyCap * 2) +
-         (kPromotedContentAdsPerDayFrequencyCap * 2));
+const int kForDays = 7;
```

Comment:
can we make this a parameter as well?

Reply:
The UI controls the amount of days, see `const int kDaysOfAdsHistory = 7;` in `brave_rewards_page_ui.cc` so would prefer this to be a future change. Thanks